components/org.apache.stratos.python.cli/src/main/python/cli/Main.py

```diff
@@ -0,0 +1,38 @@
+import sys
```

Comment:
License statement missing. You can configure most IDEs to automatically insert the statement to a new file.